src/js/gi/components/profile/CautionaryInformation.jsx

```diff
@@ -1,43 +1,52 @@
 import React from 'react';
-import { connect } from 'react-redux';
-import { showModal } from '../../actions';
+
 import AlertBox from '../../../common/components/AlertBox';
 
 const TableRow = ({ description, thisCampus, allCampuses }) => {
+  if (!thisCampus && !allCampuses) return null;
```

Comment:
`TableRow` reports the same data as `ListRow`, but the table splits the number of complaints into two columns; complaints for this campus and complaints for all campuses. So just hiding the row if both are zero.